web/src/components/shared/productItem/ProductItem.tsx

```diff
@@ -22,13 +23,26 @@ export const ProductItem = (item: Product): JSX.Element => {
             </div>
             <div className="product__body">
                 <div className="product__title">
-                    <h4>{name}</h4>
+                    {name.length > 30 ? (
```

Comment:
actually you can pass width or length to elipsis. and set logic there

Reply:
ellipsis width should be optional and default value should be 100%
width props can be either number or string so users

Reply:
styles example:
    white-space: nowrap;
    text-overflow: ellipsis;
    overflow: hidden;


width should be passed as an inline style